docker/registry/manylinux2014_x86_64/gradle.properties

```diff
@@ -0,0 +1,3 @@
+io.deephaven.project.ProjectType=DOCKER_REGISTRY
+deephaven.registry.imageName=manylinux2014_x86_64:1
+deephaven.registry.imageId=quay.io/pypa/manylinux2014_x86_64@sha256:6f0a4ff6decf82b7663bef67fb9a9de5a5047878c670235f3e96041bf0edf206
```

Comment:
I've verified that this image is only produced for amd64, so it probably needs the extra qualifier:
```
deephaven.registry.platform=linux/amd64
```